README.md

```diff
@@ -36,27 +36,27 @@ defmodule MyApp.Factory do
   # without Ecto
   use ExMachina
 
-  def factory(:user, _attrs) do
+  def factory(:user) do
     %User{
       name: "Jane Smith",
       email: sequence(:email, &"email-#{&1}@example.com"),
     }
   end
 
-  def factory(:article, attrs) do
+  def factory(:article) do
     %Article{
       title: "Use ExMachina!",
       # has_many/has_one associations are inserted when you call `create`
       comments: [build(:comment)],
       # only available in ExMachina.Ecto
```

Comment:
Maybe this can be changed to `# ExMachina.Ecto will insert the record if `create/2` is called?